README.md

```diff
@@ -1,27 +1,41 @@
 # godot-cpp
 
-**C++ bindings for the Godot script API.**
-
-The instructions below feature the new NativeScript 1.1 class structure and will only work for modules created for Godot 3.1 and later. Use the following branches for older implementations:
-
-Version | Branch
---- | ---
-**Godot 3.0 Nativescript 1.0** | [3.0](https://github.com/godotengine/godot-cpp/tree/3.0)
-**Godot 3.1 Nativescript 1.0** | [nativescript-1.0](https://github.com/godotengine/godot-cpp/tree/nativescript-1.0)
-
-## Table of contents
+This repository contains the  *C++ bindings* for the [**Godot Engine**](https://github.com/godotengine/godot)'s GDNative API.
 
+- [**Versioning**](#versioning)
 - [**Contributing**](#contributing)
 - [**Getting Started**](#getting-started)
 - [**Creating a simple class**](#creating-a-simple-class)
+- [**Updating Headers**](#updating-headers)
+
+## Versioning
+
+This repositories follows the same branch versioning as the main [Godot Engine
+repository](https://github.com/godotengine/godot):
+
+- `master` tracks the current development branch. As this is a moving target,
+  the headers in the `godot-headers` submodule may not always be fully in sync with upstream.
+  See [**Updating Headers**](#updating-headers) if you need to bring
+  them up to date.
+- `3.x` tracks the development of the next 3.x minor release. Like `master`, it
+  might not always be fully up-to-date with upstream.
+- Other versioned branches (e.g. `3.3`, `3.2`) track the latest stable release
+  in the corresponding branch.
+
+Stable releases are also tagged on this repository:
+[**Tags**](https://github.com/godotengine/godot-headers/tags).
```

Comment:
Wrong URL, and that's not true currently for `godot-cpp`.

Reply:
missed that one :)